pkg/minikube/cluster/cluster.go

```diff
@@ -585,3 +604,17 @@ func EnsureMinikubeRunningOrExit(api libmachine.API) {
 		os.Exit(1)
 	}
 }
+
+func transferAddonFileToVM(addonPath string, client *ssh.Client) error {
```

Comment:
If this takes a reader instead of a file you could collapse a lot of the logic with the other addons we transfer, from the minikube assets.